index.js

```diff
@@ -6,32 +6,45 @@
 // // => 2
 // console.log("%c----------", "color: red") 
 
+let counter =1
+counter = 2
+
+
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
 
+
+
 // console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
 // console.log("%c----------", "color: red") 
 
+const name = `michael`
+
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
-
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
-
-
+console.log("%cQuestion 3", "color: red") 
+
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
+
+function drinkWater(currentThirstLevel) {
+  console.log("Man I sure am thirsty");
+  currentThirstLevel -= 1;
+  console.log("Ahh that hits the spot");
+  return currentThirstLevel
+}
```

Comment:
perfect! you can all do currentThirsLevel-- 
Or combine lines 44 and 46 and just do return currentThirstLevel - 1